internal/harness/harness.go

```diff
@@ -0,0 +1,19 @@
+package harness
+
+import "context"
+
+var (
+	DefaultEntrypoint = []string{"/bin/sh"}
+	DefaultCmd        = []string{"-c", "tail -f /dev/null"}
```

Comment:
I have concerns about these fields. Slices are per definition mutable, and I made these into functions originally to prevent the unlikely, but possible scenario that someone might accidentally overwrite the fields and cause every other invocation of these to run with changed inputs.

Reply:
good callout, fixed!